entries/l/lihkg.com.json

```
@@ -0,0 +1,13 @@
+{
+  "LIHKG": {
+    "domain": "lihkg.com",
+    "tfa": [
+      "email",
+      "totp"
+    ],
+    "documentation": "https://help.lihkg.com/privacy/account-security/",
+    "keywords": [
+      "social"
+    ]
```

Since it's very region specific I think it would be a good idea to add the `hk` region (and possibly `cn` as well) 
```suggestion
    ],
    "regions": [
      "hk"
    ]
```

---

Actually the site is banned in China 😄